proposals/20220823-huggingface_model_pusher.md

```diff
@@ -12,25 +12,48 @@
 **Project name:** HuggingFace Model Pusher
 
 ## Project Description
-HuggingFace Model Pusher(`HFModelPusher`) pushes blessed model to the [HuggingFace Model Hub](https://huggingface.co/models).
+HuggingFace Model Pusher(`HFModelPusher`) pushes blessed model to the [HuggingFace Model Hub](https://huggingface.co/models). Also, it optionally pushes application to [HuggingFace Space Hub](https://huggingface.co/spaces).
 
 ## Project Category
 Component
 
 ## Project Use-Case(s)
-The HuggingFace Model Hub lets us have [Git-LFS](https://git-lfs.github.com) enabled repositories in public and private modes. Supported models hosted on the HuggingFace Model Hub can be directly loaded/used with APIs provided by [transformers](https://huggingface.co/docs/transformers/index) package. However, it is not limited. We can host arbitrary types of models too. 
+The HuggingFace Model and Space Hubs let us have [Git-LFS](https://git-lfs.github.com) enabled repositories in public and private modes, and they are easy to manage versions sepecially for those familiar with Git. 
```

Comment:
```suggestion
The HuggingFace Model and Space Hubs let us have [Git-LFS](https://git-lfs.github.com) enabled repositories in public and private modes. Since these repositories are primarily based on Git, they're easier as far as version control is concerned. 
```